### ISSUE TITLE
GSLB Topology cannot reference Region record

### ISSUE THREAD

**Issue by focrensh** [F5Networks/f5-appsvcs-extension#175]
##### Do you already have an issue opened with F5 support?
No

##### ISSUE TYPE

 - Bug Report

##### AS3 BUILD/ VERSION
```
3.15
```

##### BIGIP VERSION
```
14.1
```

##### SUMMARY

```
Referencing region records from topology records assumes app name in path even though it is created directly in Common.
```

##### STEPS TO REPRODUCE

```
{
    "class": "ADC",
    "schemaVersion": "3.14.0",
    "id": "test123",
    "Common": {
        "class": "Tenant",
        "Shared": {
            "class": "Application",
            "template": "shared",
            "west": {
                "class": "GSLB_Topology_Region",
                "members": [
                    {
                        "matchType": "subnet",
                        "matchOperator": "equals",
                        "matchValue": "10.20.0.0/16"
                    }
                ]
            },
            "topology": {
                "class": "GSLB_Topology_Records",
                "longestMatchEnabled": true,
                "records": [
                    {
                        "destination": {
                            "matchType": "subnet",
                            "matchValue": "1.2.3.6/32"
                        },
                        "source": {
                            "matchType": "region",
                            "matchValue": {
                                "use": "west"
                            }
                        }
                    }
                ]
            }
        }
    }
}
```

##### EXPECTED RESULTS

```
Topo record referencing the west region
```

##### ACTUAL RESULTS

```
        {
            "code": 422,
            "message": "declaration failed",
            "response": "0107025c:3: Topology record contains an nonexistent region: /Common/Shared/west",
            "host": "localhost",
            "tenant": "Common",
            "runTime": 2731
        },
```

**Comment by dstokesf5** [comment]
This is being tracked in the internal product backlog as AUTOTOOL-613.

**Comment by dstokesf5** [comment]
This was resolved in the 3.23.0 release.
